api/v1alpha1/gatekeeper_types.go

```diff
@@ -86,6 +87,19 @@ type GatekeeperSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Pod Annotations"
 	// +optional
 	PodAnnotations map[string]string `json:"podAnnotations,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Config Configuration"
+	// +optional
+	Config *ConfigConfig `json:"config,omitempty"`
+}
+
+type ConfigConfig struct {
+	// +optional
+	// This field is the same type as the gatekeeper config spec.config.match. These will be appended to config.spec.match
+	Matches []v1alpha1.MatchEntry `json:"matches,omitempty"`
+	// +optional
+	// Default is false. Setting this to true blocks adding Matches to spec.config.match.
+	DisableDefaultMatches *bool `json:"disableDefaultMatches,omitempty"`
```

Comment:
Why is this a pointer?

Reply:
To check DisableDefaultMatches exist or not `null`
It could be like:
```
Config:
   matches
        - exempt....
```